model/histogram/histogram_test.go

```diff
@@ -1107,6 +1361,88 @@ func TestHistogramValidation(t *testing.T) {
 			errMsg:    `3 observations found in buckets, but the Count field is 2: histogram's observation count should equal the number of observations found in the buckets (in absence of NaN)`,
 			skipFloat: true,
 		},
+		"rejects a normal histogram with custom buckets schema": {
```

Comment:
custom bounds:
need a testcase to reject if the first offset in positive spans is negative
also a testcase to reject if some subsequent offset is negative
also a testcase to reject if bucket count is different from what the spans cover